src/map/utils/battleutils.cpp

```diff
@@ -939,11 +942,20 @@ namespace battleutils
         if (PAttacker->getMod(Mod::ENSPELL) > 0 &&
             (PAttacker->getMod(Mod::ENSPELL_CHANCE) == 0 || PAttacker->getMod(Mod::ENSPELL_CHANCE) > tpzrand::GetRandomNumber(100)))
         {
-            static SUBEFFECT enspell_subeffects[8] = {
-                SUBEFFECT_FIRE_DAMAGE,      SUBEFFECT_ICE_DAMAGE,   SUBEFFECT_WIND_DAMAGE,  SUBEFFECT_EARTH_DAMAGE,
-                SUBEFFECT_LIGHTNING_DAMAGE, SUBEFFECT_WATER_DAMAGE, SUBEFFECT_LIGHT_DAMAGE, SUBEFFECT_DARKNESS_DAMAGE,
+            static SUBEFFECT enspell_subeffects[8] =
```

Comment:
Is there a reason we're defining this here?  This looks like duplicated code from enum SUBEFFECT (except off by one).  Totally a result of formatting and not additional code though.

Reply:
No idea, way before my time

Reply:
All element's _should_ be aligned globally now, so tables like this shouldn't be needed anymore. But that's outside of the scope of this PR 👍